app/javascript/components/public_health/PatientsTable.js

```diff
@@ -225,18 +251,20 @@ class PatientsTable extends React.Component {
 
   handleJurisdictionChange = jurisdiction => {
     if (jurisdiction !== this.state.query.jurisdiction) {
-      this.updateTable({ ...this.state.query, jurisdiction, page: 0 });
-      this.updateAssignedUsers(jurisdiction, this.state.query.scope, this.props.workflow, this.state.query.tab);
+      this.updateAssignedUsers({ ...this.state.query, jurisdiction, page: 0 });
       localStorage.removeItem(`SaraPage`);
+      localStorage.removeItem(`SaraSortField`);
+      localStorage.removeItem(`SaraSortDirection`);
       localStorage.setItem(`SaraJurisdiction`, jurisdiction);
     }
   };
 
   handleScopeChange = scope => {
     if (scope !== this.state.query.scope) {
-      this.updateTable({ ...this.state.query, scope, page: 0 });
-      this.updateAssignedUsers(this.state.query.jurisdiction, scope, this.props.workflow, this.state.query.tab);
+      this.updateAssignedUsers({ ...this.state.query, scope, page: 0 });
       localStorage.removeItem(`SaraPage`);
+      localStorage.removeItem(`SaraSortField`);
```

Comment:
Another place I'm not sure we want to clear the sort data. This can be discussed in one thread probably but I want to call out the places I'm not sure it should be.

Reply:
Yep, I agree with your comments on these lines. I will remove them in the next commit.